src/main/app/src/app/klanten/bedrijf-view/bedrijf-view.component.html

```diff
@@ -9,7 +9,7 @@
     </mat-card-header>
     <mat-card-content>
         <zac-skeleton-loader [loading$]="bedrijf$"
-                             [options]="{layout: skeletonLoaderLayout.FIELDS, fields:5, fieldsPerRow:5}">
+                             [options]="{layout: skeletonLoaderLayout.FIELDS, fields:7, fieldsPerRow:7}">
```

Comment:
Is het niet beter om meerdere rijen te gebruiken, bv [handelsnaam en adres], [kvk vestigingsnummer en type], [telefoon, email]